test/unit/mongo_client.test.js

```diff
@@ -813,4 +815,73 @@ describe('MongoOptions', function () {
       expect(client.options).to.have.property('mongoLoggerOptions').to.equal(expectedLoggingObject);
     });
   });
+
+  context('when mongodbLogPath is in options', function () {
+    const loggerFeatureFlag = Symbol.for('@@mdb.enableMongoLogger');
+
+    let stderrStub;
+    let stdoutStub;
+
+    before(() => {
```

Comment:
same comment here about preferring `beforeEach` and `afterEach`

Reply:
sounds good, I'll change it!